packages/repo-metadata-lint/src/issue-opener.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import {logger} from 'gcf-utils';
```

Comment:
Same question about `logger`.  This is why in `policy` we pass the logger to use as a constructor argument, and pass either this `logger` or `console` depending on the entry point.

Reply:
`gcf-utils` `logger` outputs to `stdout`, there's no integration with the logger API, it just outputs in a structured format that gets picked up by cloud's logging.

Siince I'm just using the CLI for debugging mainly, I don't mind the structured log output to the terminal personally.

Reply:
We use [this interface](https://github.com/googleapis/release-please/blob/d8bb7caddfa14aabd3bfa19008c10ed911638a66/src/util/logger.ts#L24-L36) in `release-please` and `code-suggester` for injecting the logger interface. gcf-util's logger and `console` both work here.

Reply:
I'm sticking with the JSON output for now, if we find this is too ugly for users happy to revisit in a follo wup PR.